pkg/apis/pipeline/v1/swagger.json

```diff
@@ -155,6 +155,10 @@
       "description": "TaskRunStepArtifact represents an artifact produced or used by a step within a task run. It directly uses the Artifact type for its structure.",
       "type": "object",
       "properties": {
+        "isBuildArtifact": {
+          "description": "A collection of values related to the artifact",
```

Comment:
This isn't a good description. We should update this.

Reply:
I think this was due to the fact that the comment was after the field instead of before so it got mapped to the next one.

Fixing!